.github/workflows/matplotlib-tests.yml

```diff
@@ -0,0 +1,75 @@
+# Run tests and integration tests on Mac, which are triggered by each master push.
+# Currently, Python3.8 is only used as an environment.
+# This is mainly for the sake of speed.
+name: matplotlib-tests
+
+on:
+  pull_request:
+    paths:
+      - tests/visualization_tests/matplotlib_tests/**
+      - .github/workflows/matplotlib-tests.yml
+      - optuna/visualization/**.py
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref == 'refs/heads/master' && github.run_number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  matplotlib-tests:
+    runs-on: ubuntu-latest
+
+    # Scheduled Tests are disabled for forks.
+    if: (github.event_name == 'schedule' && github.repository == 'optuna/optuna') || (github.event_name != 'schedule')
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
```

Comment:
I think these version updates proposed in https://github.com/optuna/optuna/pull/5253 haven't been accepted by the master branch yet. Using an older version makes this PR review smooth.